src/Layouts/Partials/FillItem.vala

```diff
@@ -217,7 +217,104 @@ public class Akira.Layouts.Partials.FillItem : Gtk.Grid {
 
         color_picker = new Gtk.Grid ();
         color_picker.get_style_context ().add_class ("color-picker");
+        color_picker.row_spacing = 12;
+
+        // Document Grid - The Grid that will have all of the document saved colors
+        var document_grid = new Gtk.Grid ();
```

Comment:
This shouldn't be a grid.
It's better to use a `Gtk.FlowBox` since we need these colors to be floating in different rows based on the space available.
This will also allow us to reorder them with drag&drop more easily.
https://valadoc.org/gtk4/Gtk.FlowBox.html

Reply:
I think you meant the `document_colors_grid` to be a FlowBox since this is what's containing the colors